src/pretalx/person/forms.py

```diff
@@ -238,6 +239,9 @@ def save(self, **kwargs):
                 setattr(self.user, user_attribute, value)
             self.user.save(update_fields=[user_attribute])
 
+            if self.user.get_gravatar:
+                gravatar_cache.apply_async(args=(self.user.pk,))
```

Comment:
Shouldn't this be dedented, and not run once for every attribute in `self.user_fields`?

Could you also check a few lines above if we still need the `get_gravatar` handling there at all?

Reply:
Oh yes, that should have been dedented.

About the gravatar handling above ... i'm not 100% sure, but i believe the field is either sent (if checked) or not sent at all when submitting the form, thus we explicitely need to set the value to `False` to unset gravatar.
Due to the immediate triggering of the caching it is highly unlikely to ever get manually un-set, but still possible (for example, if gravatar returns some error code and the user decides they want to manually upload the avatar instead)